tests/PHPStan/Analyser/data/filter-var-returns-non-empty-string.php

```diff
@@ -82,7 +82,7 @@ public function run(string $str, int $int, int $positive_int, int $negative_int)
 		assertType('9', $return);
 
 		$return = filter_var(1.0, FILTER_VALIDATE_INT, ['options' => ['min_range' => 1, 'max_range' => 9]]);
```

Comment:
I guess you could now also test `filter_var(111.0, FILTER_VALIDATE_INT, ['options' => ['min_range' => 1, 'max_range' => 9]])` that it returns `false`?

Reply:
good idea, it does luckily 😅 I do not like this extension..